test/lib/dynamo/test/lib/asserter/README.md

```
@@ -0,0 +1,33 @@
+
+![Verify Assertion Failure](https://github.com/mucbuc/asserter/workflows/Verify%20Assertion%20Failure/badge.svg)
+
+assertion util inspired by John Torjo's solution described [here](https://www.drdobbs.com/cpp/enhancing-assertions/184403745)
+
+## build
+
+see `test/build_and_run`
+
+## standard usage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)